src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,372 @@
+
+***Audio for an mp4***
+
+    ffmpeg -i myvideo.mp4 -c:v copy -c:a aac myvideo.m4a
+
+***webm/VP9***
+
+    ffmpeg -i myvideo.webm -v:c libvpx-vp9 -v:a copy myvideo.webm
+
+***Audio for a webm***
+
+    ffmpeg -i myvideo.webm -v:c copy -v:a libvorbis myvideo.webm
+    ffmpeg -i myvideo.webm -v:c copy -v:a libopus myvideo.webm
+
+## Demux (split audio and video)
+
+My preference would be to show file type conversion before diving into demuxing.
+It appears that Shaka Packager cannot do conversion without also demuxing.
+
+**mp4/Shaka Packager**
+ 
+    packager input=myvideo.mp4,stream=video,output=myvideo_video.mp4
+    packager input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+ 
+With Shaka Packager you can combine these.
+ 
+    packager \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4 \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+      
+**webm/Shaka Packager**
+
+    packager \
+      input=myvideo.webm,stream=video,output=myvideo_video.webm \
+      input=myvideo.webm,stream=audio,output=myvideo_audio.webm
+ 
+**mp4/ffmpeg**
+ 
+    ffmpeg -i myvideo.mp4 -vcodec copy -an myvideo_video.mp4
+    ffmpeg -i myvideo.mp4 -acodec copy -vn myvideo_audio.m4a
+    
+**webm/ffmpeg**
+ 
+    ffmpeg -i myvideo.webm -vcodec copy -an myvideo_video.webm
+    ffmpeg -i myvideo.webm -acodec copy -vn myvideo_audio.webm
+
+## Package
+
+### DASH/MPD
+
+    packager \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.mp4 \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4 \
+      --mpd_output myvideo_vod.mpd
+
+### HLS
+
+    ffmpeg -i myvideo.mp4 -c:a copy -b:v 8M -c:v copy -f hls -hls_time 10 \
+           -hls_list_size 0 myvideo.m3u8
+
+## Clear Key Encryption
+
+### Create a key
+
+The same method for creating a key may be used with both DASH and HLS. The
+following will create a key made of 16 hex values.
+ 
+    openssl rand 16 > media.key
+    
+This command creates a file with white space and new line characters, which are
+not allowed by Shaka Packager. You can pipe the output to some other commands
+before dumping it to a file.
+
+    openssl rand 16 | tr '\n' ' ' | sed 's/ //g' > media.key
+
+### Encrypt
+
+For the `-key` flag can be the key created earlier and stored in the media.key
+file. However, when entering it on the command line, be sure to remove the
+whitespace from the key. For the `-key_id` flag repeat the key value.
+
+> **Note:** Technically, the key ID is supposed to be either the first 8 OR the
+> first 16 hex digits of the key. Since packager requires the key to be 16
+> digits and does not allow a 32 digit key, both flags use the same value.
+
+ 
+    packager \
+      input=myvideo.mp4,stream=audio,output=glocka.m4a \
+      input=myvideo.mp4,stream=video,output=glockv.mp4 \
+      --enable_fixed_key_encryption --enable_fixed_key_decryption \
+      -key 7bbef28644876a545a40caaaad9b3200 -key_id 7bbef28644876a545a40caaaad9b3200 \
```

Comment:
Probably want to have an [INSERT KEY HERE] or similar here, otherwise folks might copy-paste the key values not realizing that they need to be unique.